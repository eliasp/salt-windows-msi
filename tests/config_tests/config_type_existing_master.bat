msiexec /i test.msi START_MINION="" MASTER=commandline1 /l*v config_type_existing_master.install.log /qb
