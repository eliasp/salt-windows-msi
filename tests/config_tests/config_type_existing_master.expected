master: commandline1
id: bob
